objects/ArkhamSCED300-1262023-Page1.f873a8.json

```diff
@@ -0,0 +1,137 @@
+{
+  "AltLookAngle": {
+    "x": 0,
+    "y": 0,
+    "z": 0
+  },
+  "Autoraise": true,
+  "ColorDiffuse": {
+    "b": 1,
+    "g": 1,
+    "r": 1
+  },
+  "Description": "- Revamped the entire table!\n- Revamped (almost) all official scenarios to use dynamic location marking!!!\n- Added an introductory tour by Joe Sargent and the Investigators of Arkham!!!!!\n- Revamped the deck construction! Cards and investigators can now be laid out by cycle as well as by class. (For now, Gloria Goldberg is with the Investigator Packs.)\n",
```

Comment:
I share the excitement, but would cut down on the exclamation mark usage ;)

Reply:
Fiiiiiine